aisdc/attacks/attack.py

```diff
@@ -14,3 +17,20 @@ def attack(self, dataset: Data, target_model: sklearn.base.BaseEstimator) -> Non
 
     def __str__(self):
         raise NotImplementedError
+
+
+class ConfigFile:  # pylint: disable=too-few-public-methods
+    """ConfigFile class to load parameters from json configuration file"""
+
+    def __init__(self, config_filename):
+        self.config_filename = config_filename
+
+    def load_config_file_into_dict(self, attack_args_dict: dict) -> dict:
+        """Reads a configuration file and loads it into a dictionary object"""
+        if self.config_filename is not None:
+            if os.path.isfile(self.config_filename):
+                with open(self.config_filename, encoding="utf-8") as f:
+                    config = json.loads(f.read())
+                for _, k in enumerate(config):
+                    attack_args_dict[k] = config[k]
+        return attack_args_dict
```

Comment:
if `self.config_filename` is None then this will crash I think? 
Propose removing the if. Object cannot be created without a filename being passed into the constructor.

Reply:
Thanks, changed as suggested.